components/dashboard/table/Filters.js

```diff
@@ -68,10 +68,10 @@ export default function Filters({
             style={{
               position: strategy,
               top: y ?? '',
-              left: x ?? '',
+              left: x>0? x??'':"14px",
```

Comment:
Please try to convert this style tag properties into tailwind classes, since they are easy conditional statements and can be converted into tailwind properties.